lib/address-derivation-discovery/lib/Cardano/Wallet/Address/Derivation/Byron.hs

```diff
@@ -242,12 +238,8 @@ hashSeed = BA.convert . cbor . blake2b256 . cbor . BA.convert
 -- derivation path. PBKDF2 encryption using HMAC with the hash algorithm SHA512
 -- is employed.
 hdPassphrase :: XPub -> Passphrase "addr-derivation-payload"
-hdPassphrase masterKey = Passphrase $
-    PBKDF2.generate
-    (PBKDF2.prfHMAC SHA512)
-    (PBKDF2.Parameters 500 32)
-    (unXPub masterKey)
-    ("address-hashing" :: ByteString)
+hdPassphrase masterKey = Passphrase $ BA.convert $
+    generateKeyByron (unXPub masterKey)
```

Comment:
I would like to keep the specification of the numerical values in `Cardano.*`. See other comment.

Reply:
sounds reasonable. done